cddl/ear-appraisal.cddl

```diff
@@ -1,6 +1,12 @@
+;# import ar4si as ar4si
+
 EAR-appraisal = {
-  ear.status => $ar4si.trust-tier
-  ? ear.trustworthiness-vector => ar4si.trustworthiness-vector
-  ? ear.appraisal-policy-id => text
+  status-label => ar4si.trust-tier
+  ? trustworthiness-vector-label => ar4si.trustworthiness-vector
+  ? appraisal-policy-id-label => text
   * $$ear-appraisal-extension
 }
+
+status-label = eat.JC<"ear.status", 1000>
+trustworthiness-vector-label = eat.JC<"ear.trustworthiness-vector", 1001>
+appraisal-policy-id-label = eat.JC<"ear.appraisal-policy-id", 1003>
```

Comment:
Why use `eat.JC<>` here but `.feature "json"` and `.feature "cbor"` above?

Reply:
Ah ok, makes sense.

Reply:
The `ar4si-autogen.cddl` in the `small-edits` branch on AR4SI does not expand anymore.

Reply:
Addressed in [c454333](https://github.com/thomas-fossati/draft-ear/pull/44/commits/c454333b27d927cb81b7f6868f97cb30cd01e991)